### PR TITLE
fixes Shadowling Glare

### PR DESCRIPTION
**Pull request by ghost**
:cl: thefastfoodguy
fix: Shadowling glares now last for 10 seconds as opposed to 1
/:cl:

whoops i forgot a 0

### CODE REVIEW

File: hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -50,8 +50,8 @@
 			to_chat(target, "<span class='userdanger'>Your gaze is forcibly drawn into [user]'s eyes, and you are mesmerized by the heavenly lights...</span>")
 		else //Only alludes to the shadowling if the target is close by
 			to_chat(target, "<span class='userdanger'>Red lights suddenly dance in your vision, and you are mesmerized by their heavenly beauty...</span>")
-		target.Stun(10)
-		M.silent += 10
+		target.Stun(100)
+		M.silent += 100
```
Comment (tortellinitony) [inline hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm]: Fairly certain silent is in seconds.

**Review by tortellinitony — CHANGES_REQUESTED** [review]

File: hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm]: fixed

**Review by tortellinitony — APPROVED** [review]

**Comment by ghost** [comment]
changelog says 10 seconds, code says 20

**Comment by ghost** [comment]
Apparently it's measured in HALF-DECISECONDS

**Review by tortellinitony — APPROVED** [review]

**Comment by PopNotes** [comment]
"HALF-DECISECONDS"
_what the fucking fuuuck?_

**Pull request merged**